src/test/java/de/tobiasroeser/lambdatest/proxy/ExampleProxyTest.java

```diff
@@ -8,6 +8,8 @@
 
 	interface Dependency {
 		String hello();
+
+		int foobar(String s1, int i2);
```

Comment:
it is relative complex to format generic return types and arguments, because of recursive structure (https://docs.oracle.com/javase/tutorial/reflect/member/methodType.html) and arrays have to be handled too. I do not have the time atm to implement this. But I think my current  implementation is:
- a little nicer then before 
- only a exception message and
- the old version is printed too.

Reply:
The thing is, all generic parameters need to be `Object`, as we need to match the generic erasure. Thus, in theory, there should not be much logic involved. Anyway, I'll merge as is.